File: Swift/Easy/Two-Sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        if nums.count < 0 {
            return []
        }
        var rValue = [Int]()
        var myDict: [Int:Int] = [:]
        
        for index in 0...nums.count - 1 {
            if let val = myDict[nums[index]] {
                rValue = [val, index]
            }
            else {
                myDict[target-nums[index]] = index
            }
        }
        return rValue
    }
}

File: Python/Easy/Two-Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        #taking care of edge case
        if len(nums) <0:
            return False 
        
        myDict = {}
        
        for index in range(len(nums)):
            
            if nums[index] in myDict:
                return myDict[nums[index]], index
                
            else:
                myDict[target-nums[index]] = index
